### DISCUSSION TITLE
Fine-tuned model is missing config.json

### DISCUSSION THREAD

**Discussion by davidress-ILW** [urchade/GLiNER#197] [Q&A]
I have been using the GLiNER training branch to fine-tune a model, and while the training succeeds, the model checkpoint folders only have the gliner_config.json and pytorch_model.bin files.

When I go to use the best checkpoint model, I receive the following error message from the from_proetrained() method:

`
config.json not found in C:\xxxx\GlinerFineTuning\vm6\model_1000
`
So, what should be in the config.json file?  Is it needed, or is GLiNER using default values from somewhere?

**Comment by shuaiL8** [comment]
Did you fix the issue? I encountered same warning message. It's probably related to the default config for Transformers, but don't know how to fix it.
